test/e2e/webdriver/driver.js

```diff
@@ -373,6 +373,12 @@ class Driver {
     await element.click();
   }
 
+  async clickElementAndWaitToDisappear(rawLocator) {
+    const element = await this.findClickableElement(rawLocator);
+    await element.click();
+    await element.waitForElementState('hidden');
```

Comment:
Would it make sense to set an explicit timeout here of the maximum time we expect the notification element to be visible in this step?

Reply:
absolutely @legobeat :bowing_woman: :heart:  I've pushed a new commit to address that